installation_docs/manual/wsl.md

```
@@ -0,0 +1,119 @@
+     sudo systemctl enable postgresql.service
+     ```
+
+
+#### Setup
+> **Note**: PostgreSQL and Redis *must* be running. PostgreSQL must be configured with a default user
+
+1. Install Ruby bundler : `gem install bundler`
+2. Install Ruby dependencies: `bundle install`
+3. Install Yarn dependencies: `yarn`
+4. Configure your PostgreSQL database in `config/database.yml` (copy `config/database.example.yml` for the template): 
+     * **(macOS/linux):** `cp config/database.example.yml config/database.yml`
```

Markdownlint (0.37.0)

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)